From def05f45dbd9aba0f67710ee3aa639d0f99f7c83 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Mon, 24 Dec 2018 00:57:43 +0200
Subject: [PATCH 20/29] extract_utils: remove the "set +e" calls during
 extract()

* There is no point in doing "set -e" in the first place if you're
  going to disable it. The subprocesses that are expected to return
  non-zero error code during normal runtime are supposed to be caught
  using the "|| :" syntax (where ":" is "true", or "no-op"), or
  "|| { custom code here }".

Change-Id: Id0c5c1e42c03cc226b39dd655c40469995f302aa
Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
---
 build/tools/extract_utils.sh | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/build/tools/extract_utils.sh b/build/tools/extract_utils.sh
index 23552037..f5864a88 100644
--- a/build/tools/extract_utils.sh
+++ b/build/tools/extract_utils.sh
@@ -930,8 +930,6 @@ function extract() {
         parse_file_list "$1" "$3"
     fi
 
-    # Allow failing, so we can try $DEST and/or $FILE
-    set +e
 
     local FILELIST=( ${PRODUCT_COPY_FILES_LIST[@]} ${PRODUCT_PACKAGES_LIST[@]} )
     local HASHLIST=( ${PRODUCT_COPY_FILES_HASHES[@]} ${PRODUCT_PACKAGES_HASHES[@]} )
@@ -1093,9 +1091,6 @@ function extract() {
         fi
 
     done
-
-    # Don't allow failing
-    set -e
 }
 
 #
-- 
2.17.1

